va/win32/va_win32.c

```diff
@@ -58,136 +44,76 @@ typedef struct _VADisplayContextWin32 {
 
 static bool TryLoadDriverNameFromRegistry(const LUID* adapter_luid, VADisplayContextWin32* pWin32Ctx)
 {
-    if (!adapter_luid)
+    if (!adapter_luid || !pWin32Ctx)
```

Comment:
Neither of of these can be NULL - so I would omit this change.

If anything, the former argument can be dropped. since it's accessible via the latter. Also I would s/Try// and fix the type handling - return boot, which gets stored as int masked bool.

Assuming you agree - the above can be a separate change, in this or separate PR.

Reply:
Actually, `pWin32Ctx` can be NULL because we forgot to check result of `calloc` here:

https://github.com/intel/libva/blob/0d971e918da2af176f158cad3948b1c88db3d55b/va/win32/va_win32.c#L279

We need to add a check for `calloc` operation. After that I agreed that check here won't make sense anymore.

Reply:
Added the null check in its own commit, and addressed the rest of the feedback from this comment in a followup commit.